src/cloudant/design_document.py

```diff
@@ -498,7 +498,12 @@ def fetch(self):
                         view_def.pop('reduce', None),
                         **view_def
                     )
-
+        # dbcopy is transformed to a new storage location, we need to de-transform
+        if 'options' in self:
+            if 'epi' in self['options']:
+                if 'dbcopy' in self['options']['epi']:
+                    for view in self['options']['epi']['dbcopy'].keys():
+                        self['views'][view]['dbcopy'] = self['options']['epi']['dbcopy'][view]
```

Comment:
I see your point - I had been thinking similar which is why I had copied it into the old location instead of moving it so it would be accessible in both. It's a bit awkward that the python dict allows direct access to set this (in either location), but we seem to do it via a `kwargs` when creating a view. I'll look into it again and see if there is a better way.